wire-testing/src/main/java/com/squareup/wire/schema/RepoBuilder.java

```diff
@@ -107,4 +109,16 @@ public String generateCode(String typeName, String profile) throws IOException {
     ClassName typeName1 = javaGenerator.generatedTypeName(type);
     return JavaFile.builder(typeName1.packageName(), typeSpec).build().toString();
   }
+
+  public String generateKotlin(String typeName) {
+    Schema schema = schema();
+    KotlinGenerator kotlinGenerator = KotlinGenerator.get(schema);
+    com.squareup.kotlinpoet.TypeSpec typeSpec =
```

Comment:
Import `TypeSpec`

Reply:
TypeSpec is already getting used from javaPoet, hence this